paasta_tools/check_chronos_jobs.py

```diff
@@ -224,7 +224,12 @@ def sensu_message_status_for_jobs(chronos_job_config, service, instance, cluster
             sensu_status = pysensu_yelp.Status.OK
             output = "Job %s%s%s is disabled - ignoring status." % (service, utils.SPACER, instance)
         else:
-            acceptable_delay = 15 * 60
+            monitoring_overrides = chronos_job_config.get_monitoring()
+            acceptable_delay = monitoring_tools.get_chronos_job_acceptable_delay(
```

Comment:
Is the use-case here that people want to give chronos more time before an alert, if so, why is `alert_after` not enough?